cmd/minikube/cmd/start_flags.go

```diff
@@ -124,6 +124,7 @@ const (
 	listenAddress           = "listen-address"
 	extraDisks              = "extra-disks"
 	certExpiration          = "cert-expiration"
+	kubeBinariesMirror      = "kube-binaries-mirror"
```

Comment:
I think this would be easier for users to find and use if it is called "binary-mirror" 

what do u think ?

Reply:
Sounds good, shorter name will be better